src/main/java/ch/naviqore/raptor/RaptorAlgorithm.java

```diff
@@ -48,4 +50,7 @@ List<Connection> routeLatestDeparture(Map<String, Integer> departureStops, Map<S
     Map<String, Connection> routeIsolines(Map<String, LocalDateTime> sourceStops, TimeType timeType,
                                           QueryConfig config);
 
+    // TODO: Discuss if this should be added to the interface (for now added for benchmark test)
+    void prepareStopTimesForDate(LocalDate date);
```

Comment:
Agree we don't need it in the interface if we want to use the explicit implementation in the service, because I would add this to the warmup of the service.